### DISCUSSION TITLE
How to hide the scroll bar at the bottom of the Zed project bar ？

### DISCUSSION THREAD

**Discussion by mrzzmrzz** [zed-industries/zed#19404] [Feature Requests]
So annoying！

<img width="447" alt="image" src="https://github.com/user-attachments/assets/60404f6c-f82b-42f2-8892-64fb736f280d">

**Comment by CharlesChen0823** [comment]
```
"project_panel" {
    "scrollbar": {
        "show": "never"
    }
}
```
will hidden vertical and horizontal scrollbar, and the bottom scrollbar display problem I had open an [pr ](https://github.com/zed-industries/zed/pull/19403)to fix this

**Reply by mrzzmrzz** [reply]
Thank you very much for your prompt response! It effectively resolved my issue, but it's a bit unfortunate that the vertical scroll bar for the code also disappears.

**Reply by CharlesChen0823** [reply]
IMO, if the pr is landed, the bottom bar is not annoying. if you want disable separately, you could send a pr to split them in setting?

**Reply by mrzzmrzz** [reply]
Great idea, but I may not be able to contribute code to the Zed project immediately, as I'm not very familiar with Rust 😢

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
